app/controllers/users_controller.rb

```diff
@@ -1,13 +1,12 @@
 class UsersController < ApplicationController
-  has_filters %w{proposals debates budget_investments comments}, only: :show
+  has_filters %w{proposals debates budget_investments comments follows}, only: :show
 
   load_and_authorize_resource
   helper_method :author?
-  helper_method :author_or_admin?
```

Comment:
@bertocq The main reason is that apparently it was not being used anywhere in the application.